Test/Mftf/Test/SeeAffirmCheckout.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Will we also add functional tests as part of this? (making sure ALA / Checkout works?) or are the MFTF tests only meant to check if the extension itself is working (saving configurations / seeing Affirm icon at checkout)?

Reply:
These test are only meant to check if the extension is installed and configured correctly.  We dont want to add API keys to check if checkout flow works